tools/generator/Java.Interop.Tools.Generator.ObjectModel/JavadocInfo.cs

```diff
@@ -210,6 +212,43 @@ static void AddNode (ICollection<string> comments, XNode node)
 			}
 		}
 
+		// Remove, replace, or decode common HTML tags to improve what is displayed in the IDE and online.
+		static string FixUpHtml (string javadocContent)
+		{
+			var replacements = new Dictionary<string, string> {
+				{ "&lt;blockquote&gt;", "" },    { "&lt;/blockquote&gt;", "" },
+				{ "&lt;cite&gt;", "<i>" },       { "&lt;/cite&gt;", "</i>" },
+				{ "&lt;code&gt;", "<c>" },       { "&lt;/code&gt;", "</c>" },
+				{ "&lt;dd&gt;", "" },            { "&lt;/dd&gt;", "" },
+				{ "&lt;dl&gt;", "" },            { "&lt;/dl&gt;", "" },
+				{ "&lt;dt&gt;", "" },            { "&lt;/dt&gt;", "" },
+				{ "&lt;em&gt;", "<i>" },         { "&lt;/em&gt;", "</i>" },
+				{ "&lt;h1&gt;", "" },            { "&lt;/h1&gt;", "" },
+				{ "&lt;h2&gt;", "" },            { "&lt;/h2&gt;", "" },
+				{ "&lt;h3&gt;", "" },            { "&lt;/h3&gt;", "" },
+				{ "&lt;h4&gt;", "" },            { "&lt;/h4&gt;", "" },
+				{ "&lt;h5&gt;", "" },            { "&lt;/h5&gt;", "" },
+				{ "&lt;h6&gt;", "" },            { "&lt;/h6&gt;", "" },
+				{ "&lt;li&gt;", "" },            { "&lt;/li&gt;", "" },
```

Comment:
…and [speaking of](https://github.com/xamarin/java.interop/pull/1125#issuecomment-1593796640) `java/text/Collator.java`, these checks may need to be case insensitive:

```
% grep -r '<li>' --include=\*.java . | wc -l
   11939
grep -r '<LI>' --include=\*.java . | wc -l 
     579
```

The all-caps `<LI>` isn't a majority of them, but it's a pretty sizable amount.  Is it ignorable?

Reply:
I've removed this in favor of a regex term for parsing code elements as suggested.